app/src/rpc/mod.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2021 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+pub mod getresource;
+pub mod keyprovider;
+
+#[cfg(feature = "ttrpc")]
+use std::collections::HashMap;
+#[cfg(feature = "ttrpc")]
+pub mod ttrpc_protocol;
+#[cfg(feature = "ttrpc")]
+pub type TtrpcService = HashMap<String, Box<dyn ::ttrpc::MethodHandler + Send + Sync>>;
```

Comment:
how about `std::collection::HashMap<String, Box<dyn ::ttrpc::MethodHandler + Send + Sync>>` to reduce one conditional compilation?